start/sw/mobile-app/mobile-installation.md

```diff
@@ -8,4 +8,20 @@ description: Creating mobile applications using Google Android Studio.
 
 # Mobile application
 
-Today wide range of smart mobile devices make it possible to create clever applications. See [Google Android Studio](/start/sw/mobile-app/mobile-devassistant.html) for a step-by-step instructions to install Android Studio and create your own ```hello world``` mobile application.
+Today wide range of smart mobile devices make it possible to create clever applications. 
+
+# Get started
```

Comment:
Please do not use multiple first-level titles (Section Headings). Use level 2 and 3 instead.